src/main/java/com/crowdin/cli/utils/console/ExecutionStatus.java

```diff
@@ -5,8 +5,8 @@
 public enum ExecutionStatus {
 
     OK("[OK] ", "\u2714  "),
-    ERROR("[ERROR] ", "\u26D4  "),
-    WARNING("[WARNING] ", "\u26A0  "),
+    ERROR("[ERROR] ", "\u274C  "),
+    WARNING("[WARNING] ", "⚠️  "),
```

Comment:
`\u26A0` ?

Reply:
Unicode \u26A0 in my console(Linux Mint) is turning into a text warning sign(⚠️), but if specify warning sign itself, then it turns into an emoji(yellow warning sign). I don't know how it works in other consoles, but I decided to left it like this.